app/views/likes/destroy.js.erb

```diff
@@ -0,0 +1,7 @@
+var existingLikes = $("#<%= dom_id(@like.photo) %>_likes");
```

Comment:
create and destroy do the same thing. consider making one file ("toggle") and reference format.js { toggle } in the controller's create and destroy methods.